File: ExcAlgoritmoProgramacao_JS/Lista_Bee/1048_basico.js
import { question } from "readline-sync";

function main() {
    const salario = get_number();

    const reajuste = Calcular_Reajuste(salario);

    print(`Novo salario: ${reajuste[0].toFixed(2)}`);
    print(`Reajuste ganho: ${reajuste[1].toFixed(2)}`);
    print(`Em percentual: ${reajuste[2]} %`);

}

function get_number() {
    return parseFloat(question());
}

function print(texto) {
    return console.log(texto);
}

function Calcular_Reajuste(salario) {
    let novo_salario;
    let percentil;
    let percentual;

    if (salario > 0 && salario <= 400) {
        percentil = 15;
        percentual = salario * (15 / 100);
        novo_salario = (salario + percentual);
    } else if (salario <= 800) {
        percentil = 12;
        percentual = salario * (12 / 100);
        novo_salario = (salario + percentual);
    } else if (salario <= 1200) {
        percentil = 10;
        percentual = salario * (10 / 100);
        novo_salario = (salario + percentual);
    } else if (salario <= 2000) {
        percentil = 7;
        percentual = salario * (7 / 100);
        novo_salario = (salario + percentual);
    } else {
        percentil = 4;
        percentual = salario * (4 / 100);
        novo_salario = (salario + percentual);
    }
    return [novo_salario, percentual, percentil];
}

main();

File: ExcAlgoritmoProgramacao_JS/Lista_Bee/1048_basico.py
x = float(input())

r = 0.0
p = ''

if (x >= 0 and x <= 400):
    r = 0.15; p = '15 %'
elif (x <= 800):
    r = 0.12; p = '12 %'
elif (x <= 1200):
    r = 0.1; p = '10 %'
elif (x <= 2000):
    r = 0.07; p = '7 %'
else:
    r = 0.04; p = '4 %'

print(f"Novo salario: {(x + (x * r)):.2f}")
print(f"Reajuste ganho: {(x * r):.2f}")
print(f"Em percentual: {p}")